src/documents/blog/07-haxeflixel-mechanics.md

```diff
@@ -0,0 +1,19 @@
+```
+title: "HaxeFlixel Mechanics"
+layout: blog-post
+postDate: 03-02-2017
+```
+
+Does this sound familiar? "I wish I could remember how to do this one, small thing in HaxeFlixel, and that I didn't have to dig through a massive tutorial just to jog my memory." Or, how about: "What's the 'right' way to handle this, again?" Yeah, we've all been there! Wouldn't it be nice if there was some kind of interactive repository of just about every conceiveable useful snippet of code that you could use to answer these, and other questions?
+
+Well, what if I told you that it *already exists*!?
+
+A couple of years back now (wow, time flies!), I had the idea of putting together a website that would showcase a ton of little samples of HaxeFlixel code, along with a demonstration of that code in action. It took some time, but I was able to get a pretty good start on this thing, and I called it *HaxeFlixel Mechanics*. The plan was to get a bunch of samples in the project, and then invite collaborators to add more to help grow and flesh it out... and then Real Life&trade; got in the way and the project got pushed aside...
+
+This year, let's work together to get HFMechanics back up and running again. Let's add more demoes and snippets, and try to fill it with lots more good stuff.
+
+If you'd like to lend a hand, join us over in (the repository)[https://github.com/HaxeFlixel/haxeflixel-mechanics]. Add an issue for suggestions or requests, or add a PR for snippets you've created so we can add them to the site!
```

Comment:
I think "submit" a PR sounds better than "add". Also, I wouldn't use the abbreviation, some people might not be familiar with it.